storage/imxrt-flash/fspi.c

```diff
@@ -29,6 +29,9 @@
  */
 
 
+#define QUICK_POLL_MAX_RETRIES 5
```

Comment:
```suggestion
#ifndef QUICK_POLL_MAX_RETRIES
#define QUICK_POLL_MAX_RETRIES 5
#endif
```
Check if accessible from `boar_config.h`